index.html

```diff
@@ -3,9 +3,25 @@
   <head>
     <meta charset="utf-8">
     <link rel="stylesheet" href="/css/index.css">
```

Comment:
The href should be `"./css/index.css"` as there is an issue with loading the stylesheet for html file.

Reply:
I see, thanks!